CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(OpenTTD)
+if (NOT BINARY_NAME)
+    set(BINARY_NAME openttd)
+endif (NOT BINARY_NAME)
+
+project(${BINARY_NAME})
```

Comment:
Is this necessary? Shouldn't the `set_target_properties` call below do the work?

Reply:
GNUInstallDirs uses `PROJECT_NAME` for default `CMAKE_INSTALL_DOCDIR` (see https://cmake.org/cmake/help/v3.5/module/GNUInstallDirs.html#module:GNUInstallDirs)

Reply:
So then is `set_target_properties` unnecessary?

Reply:
`set_target_properties` is to rename the output of `openttd` target, created via `add_executable()`. It's unrelated to `project()`